web/src/components/dashboard/HideableTasks.tsx

```diff
@@ -39,11 +39,7 @@ export const HideableTasks = (): ReactElement => {
       <div className={`${isTabletAndUp ? "flex flex-align-center" : ""} margin-bottom-205`}>
         <h2 className="margin-bottom-0 text-medium">{Config.dashboardDefaults.upAndRunningTaskHeader}</h2>
         <div className={`mla ${isTabletAndUp ? "" : "margin-top-2"}`}>
-          <UnStyledButton
-            style={"transparentBgColor"}
-            className={"usa-tag text-normal text-base border-1px border-base-light hide-unhide-button"}
-            onClick={handleToggleClick}
-          >
+          <QuaternaryButton size={"regular"} onClick={handleToggleClick}>
```

Comment:
[Dust] When I remove the content from both the secondary button component and the QuaternaryButton button component, it results in a container with a border and padding. I believe both of these buttons should be categorized as secondary buttons due to their similar styling. The elements that are passed into the button component should not dictate the name of the component.

If you decide to keep the button, I suggest creating a 'buttons' folder to centralize all the various button components we use throughout the application.

secondary
<img width="617" alt="Screenshot 2023-09-28 at 3 36 14 PM" src="https://github.com/newjersey/navigator.business.nj.gov/assets/72581979/089ea82d-7002-4665-95f4-8a75db86b237">

quaternary
<img width="49" alt="Screenshot 2023-09-28 at 3 34 08 PM" src="https://github.com/newjersey/navigator.business.nj.gov/assets/72581979/5451515a-c250-42e7-a2b7-f984660d28a2">